src/help.h

```diff
@@ -444,7 +444,7 @@ struct commandHelp {
     9,
     "1.0.0" },
     { "GEOADD",
-    "key longitude latitude member [longitude latitude member ...]",
+    "key [CH] [NX|XX] longitude latitude member [longitude latitude member ...]",
```

Comment:
No need for this change - the file is auto-generated before a release from the commands.json file in the redis-doc repository.